docs/site/Relation-generator.md

```diff
@@ -72,6 +77,31 @@ Check the site [Relations](HasMany-relation.md) and the
 [Querying Related Models](HasMany-relation.md#querying-related-models) section
 in each relation for more use cases.
 
+#### HasManyThrough
+
+Defining a HasManyThrough relation via the CLI is a bit different from defining
+HasMany, HasOne, or BelongsTo relations.
+
+```sh
+lb4 relation --sourceModel=<sourceModel>
+--destinationModel=<destinationModel> --throughModel=<throughModel>
+--relationType=<hasManyThrough> [--relationName=<relationName>]
+[--sourceKeyOnThrough=<sourceKeyOnThrough>] [--targetKeyOnThrough<targetKeyOnThrough>]
```

Comment:
Assuming the last 3 flags are optional, just wondering what would be the default values. Perhaps it worths putting in the docs as well? Just a suggestion.

Reply:
Since we introduce the default value in the Interactive Prompts section, I feel like it would be a bit heavy to have them here again. I added `(please check the default values for optional args below)` and also a screenshot about the change. WDYT?

Reply:
Sorry that I probably missed that you have mentioned the default value. 
I think we don't need `(please check the default values for optional args below)`. :)